File: API/LuminaAPI/Model/PMS/PADetail.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace LuminaAPI.Model.PMS
{
    public class PADetail
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string _id { get; private set; }

        public string ProductID { get; set; }

        public int CategoryID { get; set; }

        public int SizeID { get; set; }

        public int ColorID { get; set; }

        public string Image { get; set; }

        public bool IsActive { get; set; }

        public int TagID { get; set; }
    }
    }

    public class ProductList
    {
        private string productID;
        private string image;
        private int price;

        public string ProductID
        {
            get { return this.productID; }
            set { this.productID = value; }
        }

        public string Image
        {
            get { return this.image; }
            set { this.image = value; }
        }

        public int Price
        {
            get { return this.price; }
            set { this.price = value; }
        }

    }

File: UI/LuminaUI/src/app/shared/Models/PaDetail.ts
export class PADetail {
    public _id: string;
    public padID: number;
    public productID: number;
    public categoryID: number;
    public sizeID: number;
    public colorCode: number;
    public alias: string;
    public image: string;
    public isActive: boolean;
  
    constructor(
      _id: string,
      padID: number,
      productID: number,
      categoryID: number,
      sizeID: number,
      colorCode: number,
      alias: string,
      image: string,
      isActive: boolean
    ) {
      this._id = _id;
      this.padID = padID;
      this.productID = productID;
      this.categoryID = categoryID;
      this.sizeID = sizeID;
      this.colorCode = colorCode;
      this.alias = alias;
      this.image = image;
      this.isActive = isActive;
    }
  }
  